src/webots/control/WbLanguageTools.cpp

```diff
@@ -124,7 +130,7 @@ QString WbLanguageTools::pythonCommand(QString &shortVersion, const QString &com
   if (pythonCommand == "!")
     WbLog::warning(QObject::tr("Python was not found.\n") + advice);
 #else  // __linux__
-    shortVersion = checkIfPythonCommandExist(pythonCommand, env, true);
+      shortVersion = checkIfPythonCommandExist(pythonCommand, env, true);
```

Comment:
This indentation looks strange to me, but atom auto-correct like that so I guess it is right

Reply:
Yes, I had to do this to pass the clang-format test... But I agree, this looks wrong...